src/main/webapp/deadlineExceededErrorPage.jsp

```diff
@@ -9,7 +9,7 @@
     <title>TEAMMATES</title>
     <link rel="stylesheet" href="/bootstrap/css/bootstrap.min.css" type="text/css"/>
     <link rel="stylesheet" href="/bootstrap/css/bootstrap-theme.min.css" type="text/css"/>
-    <link rel="stylesheet" href="stylesheets/teammatesCommon.css" type="text/css">
+    <link rel="stylesheet" href="/stylesheets/teammatesCommon.css" type="text/css">
```

Comment:
Yes, that's better. how did you come across this anyway? Just noticed the
missing slash accidentally?